templates/docs/examples/patterns/side-navigation/_toggle_script.js

```diff
@@ -6,16 +6,32 @@
 */
 function toggleDrawer(sideNavigation, show) {
   if (sideNavigation) {
+    const outsideDrawer = sideNavigation.querySelector('.p-side-navigation__toggle');
+    const insideDrawer = sideNavigation.querySelector('.p-side-navigation__toggle--in-drawer');
```

Comment:
While we are using these buttons directly I guess it would be good for them to have `aria-controls` and `aria-expanded` set to correct values.

Reply:
I'm taking a look at this now with JS (both buttons already have the aria-controls attribute). I'm wondering if we could add the `aria-expanded` attributes with HTML, as the `.p-side-navigation__toggle--in-drawer` button will only be visible when the drawer is expanded and the `.p-side-navigation__toggle` will only be visible when the drawer is collapsed